tests/api/test_payment.py

```diff
@@ -541,8 +546,15 @@ def set_braintree_customer_id(customer_user, braintree_customer_id):
     return customer_user
 
 
+@pytest.fixture
+def set_dummy_customer_id(customer_user, dummy_customer_id):
```

Comment:
Where do you use it?

Reply:
`test_list_payment_sources`

`customer_user` is used in both `set_dummy_customer_id` and `user_api_client`
that's why I don't have to explicitly use this to retrieve customer_user